meson.build

```diff
@@ -987,6 +992,13 @@ else
 endif
 conf.set10('HAVE_PWQUALITY', have)
 
+if not have and want_passwdqc != 'false' and not skip_deps
+        have = cc.has_header('passwdqc.h', required : want_passwdqc == 'true')
```

Comment:
i am sorry, but this project has no pkg-config file in 2021? This kinda suggests it's abandonware, and makes me kinda inclined not to merge this.

Checking for header files like this might be OK for libc headers and such super-low-level stuff, but come on, in 2021 normal projects shouldn't work that way.

Reply:
Will work on pkg-config file for passwdqc.